code/modules/food/beverages/bottler.dm

```diff
@@ -0,0 +1,448 @@
+				if(G.seed && G.seed.kitchen_tag == recipe.obj_2_kitchen_tag)
+					number_matches++
+			else
+				number_matches++
+		if(istype(obj_3, recipe.obj_3_item))
+			if(istype(obj_3, /obj/item/weapon/reagent_containers/food/snacks/grown))
+				var/obj/item/weapon/reagent_containers/food/snacks/grown/G = obj_3
+				if(G.seed && G.seed.kitchen_tag == recipe.obj_3_kitchen_tag)
+					number_matches++
+			else
+				number_matches++
+		if(number_matches == 3)
+			return recipe
+	return null
+
+/obj/machinery/bottler/proc/dispense_empty_container(var/container = 0)
+	switch(container)
+		if(1)	//glass bottle
+			if(glass_bottles > 0)
+				new /obj/item/weapon/reagent_containers/food/drinks/cans/bottler/glass_bottle(src.loc)
+				glass_bottles--
+		if(2)	//plastic bottle
+			if(plastic_bottles > 0)
+				new /obj/item/weapon/reagent_containers/food/drinks/cans/bottler/plastic_bottle(src.loc)
+				plastic_bottles--
+		if(3)	//can
+			if(cans > 0)
+				new /obj/item/weapon/reagent_containers/food/drinks/cans/bottler/metal_can(src.loc)
+				cans--
+
+/obj/machinery/bottler/proc/process_ingredients(var/container = 0)
+	//stop if we have ZERO ingredients (what would you process?)
+	if(!obj_1 && !obj_2 && !obj_3)
+		visible_message("<span class='warning'>There are no ingredients to process! Please insert some first.</span>")
+		return
+	//prep a container
+	var/obj/item/weapon/reagent_containers/food/drinks/cans/drink_container
+	var/container_type = ""
+	switch(container)
+		if(1)	//glass bottle
+			container_type = "Glass Bottle"
+			if(glass_bottles > 0)
+				drink_container = new /obj/item/weapon/reagent_containers/food/drinks/cans/bottler/glass_bottle()
+				glass_bottles--
+		if(2)	//plastic bottle
+			container_type = "Plastic Bottle"
+			if(plastic_bottles > 0)
+				drink_container = new /obj/item/weapon/reagent_containers/food/drinks/cans/bottler/plastic_bottle()
+				plastic_bottles--
+		if(3)	//can
+			container_type = "Can"
+			if(cans > 0)
+				drink_container = new /obj/item/weapon/reagent_containers/food/drinks/cans/bottler/metal_can()
+				cans--
+	if(!drink_container)
+		if(container_type)
+			visible_message("<span class='warning'>Error 503: Out of [container_type]s.</span>")
+		else
+			visible_message("<span class='warning'>Error 404: Drink Container Not Found.</span>")
+		return
+	//select and process a recipe based on inserted ingredients
+	visible_message("<span class='notice'>\The [src] hums as it processes the ingredients...</span>")
+	bottling = 1
+	var/datum/bottler_recipe/recipe_to_use = select_recipe()
+	if(!recipe_to_use)
+		//bad recipe, ruins the drink
+		var/contents = pick("thick goop", "pungent sludge", "unspeakable slurry", "gross-looking concoction", "eldritch abomination of liquids")
+		visible_message("<span class='warning'>The [container_type] fills with \an [contents]...</span>")
+		drink_container.reagents.add_reagent(pick("????", "toxic_slurry", "meatslurry", "glowing_slury", "fishwater"), pick(30, 50))
```

Comment:
"glowing_slurry"

Also, this would be the first source of glowing slurry added to the game. It's basically more powerful unstable mutagen, which doesn't seem like something players should be able to accidentally expose themselves to.